### ISSUE TITLE
Fixing namespace 'InputSystem' does not exist in namespace 'CoreServices' when using Assembly Definition and Unity 2020.3.11f

### ISSUE THREAD

**Issue by killerantz** [microsoft/MixedRealityToolkit-Unity#9990]
## Describe the problem
```C#
using Microsoft.MixedReality.Toolkit;
using Microsoft.MixedReality.Toolkit.Input;
using UnityEngine;

public class GetMRTKScoreServiceInputSystem : MonoBehaviour
{
    private void Start()
    {
        IMixedRealityInputSystem inputSystem = CoreServices.InputSystem;
    }
}
```
The code above causes the following error

```
Assets\CoreServiceTest\GetMRTKScoreServiceInputSystem.cs(11,48): error CS0234: The type or namespace name 'InputSystem' does not exist in the namespace 'CoreServices' (are you missing an assembly reference?)
```

1. Create a new project in Unity 2020.3.11f1
2. Import MRTK
3. Create the script like the one above
4. Add an Assembly Definition to the directory with the script
5. Add MRTK.Core as an Assembly Definition Reference and Apply
6. Observe the error from above.

Unity added a namespace called CoreServices that's now conflicting with MRTK CoreServices.

## Describe the solution you'd like

Consider changing the name to CoreServices that is less ambiguous or provide documentation on best practices when using Assembly Definitions.

## Describe alternatives you've considered

Using the full path to Microsoft.MixedReality.Toolkit.CoreServices fixes the issue, but not ideal.

An Alias is much nicer.

```C#
using MRTKCoreServices = Microsoft.MixedReality.Toolkit.CoreServices;

//

IMixedRealityInputSystem inputSystem = MRTKCoreServices.InputSystem;
```

## Additional context

Assembly Definition References

![image](https://user-images.githubusercontent.com/22085553/122292223-984edf00-ceaa-11eb-9b94-47ed21a04950.png)

**Comment by keveleigh** [comment]
@killerantz This is unfortunately a bug with Unity's Version Control / Collab package. They're accidentally including some [Xamarin types](https://docs.microsoft.com/dotnet/api/coreservices?view=xamarin-mac-sdk-14). Last I heard, they've acknowledged the issue and are looking into a fix.
Another workaround until then is to uninstall that package if you aren't using it.

**Comment by keveleigh** [comment]
Closing this as fixed in the 1.13.5 release of Unity's Version Control package: https://docs.unity3d.com/Packages/com.unity.collab-proxy@1.17/changelog/CHANGELOG.html#1135---2021-09-27
